parts/kubernetesmasteraddons-heapster-deployment.yaml

```diff
@@ -40,11 +40,9 @@ spec:
         - "--extra-cpu=0.5m"
         - "--memory=140Mi"
         - "--extra-memory=4Mi"
-        - "--threshold=5"
```

Comment:
I just saw https://github.com/Azure/acs-engine/pull/1076. I still think fixing `acs-engine` to work with `addon-resizer:v2.0` is better solution compared to keep using `addon-resizer:v1.7`

If we are using `kubernetes:1.7` and `heapster:v1.4.0`, it makes sense to use `addon-resizer:v2.0`.

https://github.com/kubernetes/kubernetes/commit/63ccedcfa7a34ba4bed8b697a07eab9e1e14896d